lib/pdksync.rb

```diff
@@ -44,10 +47,147 @@ module PdkSync
   @default_pdksync_label = Constants::PDKSYNC_LABEL
   @git_platform = Constants::GIT_PLATFORM
   @git_base_uri = Constants::GIT_BASE_URI
+  @jenkins_platform = Constants::JENKINS_PLATFORM
```

Comment:
You will need to refactor the configuration code here and add to the new configure class that was just merged in.

Reply:
Hey @logicminds , thank you. we'll update it.